### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
[DEMO LINK](https://Zelinskyi-Serhii.github.io/react_person/)

### CODE REVIEW

**Review by gretoq — CHANGES_REQUESTED** [review]

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,37 @@
...
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {`${partnerInfo}`}
```
Comment (gretoq) [inline src/components/Person/Person.jsx:33]:
```suggestion
        {partnerInfo}
```

**Review by gretoq — APPROVED** [review]
Check my suggestion

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,37 @@
...
+    partnerName,
+  } = props.person;
+
+  const isPartner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  const partnerInfo = isMarried
+    ? `${partnerName} is my ${isPartner}`
+    : `I am not married`;
```
Comment (sergii-nosachenko) [inline src/components/Person/Person.jsx:18]:
```suggestion
    : 'I am not married';
```

**Review by sergii-nosachenko — APPROVED** [review]
Good work 👍

**Pull request closed without merging**